flink-cdc-connect/flink-cdc-source-connectors/flink-connector-postgres-cdc/src/main/java/org/apache/flink/cdc/connectors/postgres/table/PostgreSQLDeserializationConverterFactory.java

```diff
@@ -96,4 +101,36 @@ public Object convert(Object dbzObj, Schema schema) throws Exception {
                     }
                 });
     }
+
+    private static Optional<DeserializationRuntimeConverter> createArrayConverter() {
```

Comment:
This is a more general way, there is no need to implement only in PostgreSQLDeserializationConverterFactory. Java
Inside, in DebeziumEventDeserializationSchema achieve more appropriate